st2api/st2api/controllers/v1/action_views.py

```diff
@@ -144,6 +143,9 @@ def get_all(self, exclude_attributes=None, include_attributes=None, sort=None, o
         result = []
         for item in resp.json:
             action_api = ActionAPI(**item)
+            # TODO: This is extreme inefficient and results in N * 2 queries where N is number of
```

Comment:
This is simply horrible, with aws and some other packs API endpoint takes 30+ seconds to return (well, now 10 seconds less with logging fix, but still too long).